application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -483,6 +483,27 @@
 
 .count {
   color: $theme.textSecondaryColor;
+}
+@media (min-width: 768px) {
+  .inline-input-row>.col-sm-4>dl {
```

Comment:
```suggestion
  .inline-input-row > .col-sm-4 > dl {
```

The selector is more readable if you leave a space before and after each "operator". In any case, this is our CSS "code" style.

Moreover, it feels unnatural to start from the bottom of the tree when writing the styles. I find it easier to read and understand the styles when they are specified from the top of the tree down the bottom. In other words, it's clearer if you specify the styles of the parent nodes first and then the styles for their descendants. This is especially important because the parent styes are inherited by the descendant nodes so it's best to look at the "base" styles first and then at the more specific (lower level) styles.